.golangci.yml

```diff
@@ -19,12 +19,12 @@ linters:
   - bidichk
   - bodyclose
   - containedctx
+  - copyloopvar
   - dogsled
   - dupword
   - durationcheck
   - errcheck
   - errchkjson
-  - exportloopref
```

Comment:
We don't want this linter anymore, because Go Team has resolved this issue by [this](https://go.dev/blog/loopvar-preview)